README.md

```diff
@@ -29,7 +29,8 @@ A source is getting associated with a consumer group ID the first time the `Read
 | name                 | description                                                                                                                                                                                                  | required | default value             |
 |----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------------------|
 | `servers`            | Servers is a list of Kafka bootstrap servers, which will be used to discover all the servers in a cluster.                                                                                                   | true     |                           |
-| `topic`              | Topic is the Kafka topic from which records will be read.                                                                                                                                                    | true     |                           |
+| `topics`             | Topics is a comma separated list of Kafka topics from which records will be read, ex: "topic1,topic1".                                                                                                       | true     |                           |
+| ~~`topic`~~          | **{{will be deprecated by `v1.0`}}** Topic is the Kafka topic to read from.                                                                                                                                  | false    |                           |
```

Comment:
I'd change the deprecation warning (the field is already deprecated), move it to the end and point the user towards the field they should be using.
```suggestion
| `topics`             | Topics is a comma separated list of Kafka topics from which records will be read, ex: "topic1,topic2".                                                                                                       | true     |                           |
| ~~`topic`~~          | Topic is the Kafka topic to read from. **Deprecated: use `topics` instead.**                                                                                                                                 | false    |                           |
```